opencti-platform/opencti-graphql/src/manager/pirManager.ts

```diff
@@ -165,29 +166,22 @@ const processStreamEventsForPir = (context:AuthContext, pir: BasicStoreEntityPir
  * Handler called every {PIR_MANAGER_INTERVAL} and studying a range of stream events.
  */
 const pirManagerHandler = async () => {
-  const redisClient = await createRedisClient(PIR_MANAGER_LABEL, false);
```

Comment:
i'm not sure to understand why we didn't pass the client anymore, and have a specific logic in redis-stream to handle the PIR case

that's said i'm sure also to understand why initialy we need a separate client for PIR ?